protocol/v2/qbft/instance/instance.go

```diff
@@ -162,7 +161,14 @@ func (i *Instance) BaseMsgValidation(msg *specqbft.SignedMessage) error {
 		return errors.Wrap(err, "invalid signed message")
 	}
 
-	if msg.Message.Round < i.State.Round {
+	// If a node gets a commit quorum before round change and other nodes get it after,
+	// then the node with quorum before wouldn't be able to receive next round messages from others
```

Comment:
don't get the comment...
a "node with [commit] quorum before" finishes consensus and doesn't need to receive next round messages.

you meant the other way around, of letting nodes that changed round receive commit messages

Reply:
I modified the comment, does it look ok now?